packages/core/test/test-util/destination-service-mocks.ts

```diff
@@ -12,8 +20,14 @@ export function mockInstanceDestinationsCall(response: any, responseCode: number
     .reply(responseCode, response);
 }
 
-export function mockSubaccountDestinationsCall(response: any, responseCode: number, accessToken: string, uri: string = destinationServiceUri) {
-  return nock(uri, {
+export function mockSubaccountDestinationsCall(
+  abc: nockFunction,
+  response: any,
+  responseCode: number,
+  accessToken: string,
+  uri: string = destinationServiceUri
+) {
+  return abc(uri, {
```

Comment:
`abc` is not the best of names. What is it?

Reply:
was a temp name renamed after the draft stage was left.